screening-schema.json

```diff
@@ -323,7 +323,7 @@
             },
             "ensurePrivacySecurity": {
               "type": "string",
-              "description": "If yes - does the project ensure the privacy and security of this data and/or content and has it taken steps to prevent adverse impacts resulting from its collection, storage and distribution.",
+              "description": "If yes - does the project ensure the privacy and security of this data and/or content collection and has it taken steps to prevent adverse impacts resulting from its collection, storage and distribution.",
```

Comment:
@prajectory I reverted the change you made from dataset to data because this question relates to PII data collected and not datasets per se. Thanks.